client/src/components/Header/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Logo from '../../assets/icons/HeaderLogo.png';
+import {Appbar, Avatar, Provider as PaperProvider} from 'react-native-paper';
+import {View} from 'react-native';
+import styles from './style';
+
+function Header() {
+  return (
+    <PaperProvider>
+      <Appbar.Header style={styles.bottom}>
+        <View style={styles.container}>
```

Comment:
You don't need the `View` container, you already have the `Appbar.Header`, use it as the main container and remove the View and it's style completely.